qiita_pet/handlers/study_handlers/sample_template.py

```diff
@@ -57,3 +61,50 @@ def post(self):
         else:
             raise HTTPError(400, 'Unknown sample template action: %s' % action)
         self.write(result)
+
+
+def build_sample_summary(study_id, user_id, visible=None):
```

Comment:
A generator + websocket would be pretty cool
On Jan 21, 2016 10:10 PM, "Antonio Gonzalez" <notifications@github.com>
wrote:

> In qiita_pet/handlers/study_handlers/sample_template.py
> <https://github.com/biocore/qiita/pull/1589#discussion_r50464520>:
>
> > @@ -57,3 +61,50 @@ def post(self):
> >          else:
> >              raise HTTPError(400, 'Unknown sample template action: %s' % action)
> >          self.write(result)
> > +
> > +
> > +def build_sample_summary(study_id, user_id, visible=None):
>
> Good point, you only need which samples are in which prep ...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/biocore/qiita/pull/1589/files#r50464520>.
>